Python/books.md

```diff
@@ -0,0 +1,3 @@
+Name | Links 
+------------ | ------------- 
+Introduction to computer science using Python - One of the best books for beginners. | http://it-ebooks.info/book/2466/

```

Comment:
I didn't get you by "Original url". I have provided url from anyone can download ebook as pdf.